### ISSUE TITLE
Please add support for default SQLite date/time format

### ISSUE THREAD

**Issue by dswhite42** [scrapinghub/dateparser#224]
SQLite stores text dates in YYYY-MM-DD HH:MM:SS.SSS format.  It's the decimal fraction that throws dateparser off:

    >>> type(dateparser.parse('2016-08-30 12:30:45'))
    <class 'datetime.datetime'>
    >>> type(dateparser.parse('2016-08-30 12:30:45.060325'))
    <class 'NoneType'>

Can this support be added?

Thanks!

**Comment by sixhobbits** [comment]
I am pretty sure that this is a bug. Version 3.4 still parsed the example given above correctly ('2016-08-30 12:30:45.060325'). However, using Version 4.0, the fractional seconds must be limited to two decimal points or omitted entirely in order to achieve a successful parse.

**Comment by waqasshabbir** [comment]
This was already fixed a week after `0.4.0` was released. See 9f5caee. However, there's still a limit on number of decimal places (micro = 6).

```
In [2]: parse('2016-08-30 12:30:45.060325')
Out[2]: datetime.datetime(2016, 8, 30, 12, 30, 45, 60325)

In [3]: parse('2016-08-30T12:30:45.060325')
Out[3]: datetime.datetime(2016, 8, 30, 12, 30, 45, 60325)
```

I tried to eliminate the *limit* but it reduced accuracy.

**Comment by dswhite42** [comment]
For use with SQLite, 6 decimal places will be great.  Thanks, Waqas - I'll look forward to 0.4.1 whenever it's released.

**Comment by waqasshabbir** [comment]
fixed #200

**Comment by dswhite42** [comment]
Many thanks!
